gin-gorm-filter.go

```diff
@@ -97,13 +97,26 @@ func filterField(field reflect.StructField, phrase string) clause.Expression {
 	} else {
 		paramName = columnName
 	}
-	re, err := regexp.Compile(fmt.Sprintf(`(?m)%v:(\w{1,}).*`, paramName))
+	re, err := regexp.Compile(fmt.Sprintf(`(?m)%v([:<>!=]{1,2})(\w{1,}).*`, paramName))
```

Comment:
I would suggest using explicit group not to handle unsupported operator combinations.
```suggestion
	re, err := regexp.Compile(fmt.Sprintf(`(?m)%v(:|!=|>|<|>=|<=)(\w{1,}).*`, paramName))
```

Reply:
Ah, that's fair.  I'll make the change.

Reply:
Done!